src/firefly/js/ui/PositionFieldDef.js

```diff
@@ -89,10 +105,15 @@ export function formatPosForHelp(wp) {
             ' &nbsp;&nbsp;&nbsp;<i>or</i> &nbsp;&nbsp;&nbsp;' +
             hmsRa + ',&nbsp;' + hmsDec + '&nbsp;&nbsp;' + csys +
             '</div>';
+
+        if (wp.getCoordSys().toString() != CoordinateSys.EQ_J2000.toString()) {
+            s += getEQJ2000(wp);
+        }
     }
     else {
-        s = '<div class=on-dialog-help>' +
+        s = '<div class=\"on-dialog-help faded-text\" style=\"font-size:10px;\">' +
```

Comment:
same here template literal (backquote) strings, The original code was written before they existed.